### PR TITLE
Add pypeln

### PR DESCRIPTION
**Pull request by pinin4fjords**
Added recipe for pypeln, built using the pypi skeleton with some missing packages added to deps. 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pinin4fjords** [comment]
Hi @conda-forge/help-python - can I get review?

File: recipes/pypeln/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry
+    - stopit >=1.1.2,<2.0.0
```
Comment (xhochy) [inline recipes/pypeln/meta.yaml:21]: Probably not needed in host
```suggestion
```

File: recipes/pypeln/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - pypeln
+    - pypeln.process
+    - pypeln.process.api
+    - pypeln.sync
+    - pypeln.sync.api
+    - pypeln.task
+    - pypeln.task.api
+    - pypeln.thread
+    - pypeln.thread.api
```
Comment (xhochy) [inline recipes/pypeln/meta.yaml:38]: `pip check` is always helpful
```suggestion
    - pypeln.thread.api
  requires:
    - pip
   commands:
    - pip check
```

**Comment by pinin4fjords** [comment]
Thanks @xhochy for review- will check out grayskull.

File: recipes/pypeln/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c9a3b72f981441e0ea7576cf54c4cd77289b6a2322497da43581cf9b47999ff4"
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/pypeln/meta.yaml:13]: As this package isn't platform-specific, this can be `noarch`.
```suggestion
  number: 0
  noarch: python
```

**Pull request merged**